### ISSUE TITLE
Increase request limits for POST / PUT datasets

### ISSUE THREAD

**Issue by jnehring** [freme-project/Broker#142]
* 200 MB
* 1 hour

**Comment by jnehring** [comment]
I wrote some tests and commited them to the "timeout test branch" in the broker repo.

* There is a new endpoint "/e-test" that takes the POST body and writes it to a file
* There is a test client in "eu.freme.broker.Test" that reads a local file (in my tests of 100 MB) and sends it to the /e-test endpoint. Every 5 MB it waits for 10 seconds

When I start first the broker and then the test client then it sends the file to the test endpoint. The request takes 200 seconds and submits a 100 MB file.

So I think we do not have to increase the request limits because they are already very high.

**Comment by m1ci** [comment]
> When I start first the broker and then the test client then it sends the file to the test endpoint. The request takes 200 seconds and submits a 100 MB file.

Yes, you managed to submit 100 MB in 200 seconds since you have relatively fast internet. The internet speed of some clients might be low as 50-100 KB/sec

IMO, we still need to consider increasing the time and size limit for processing requests.

**Comment by jnehring** [comment]
I think that right now there a maximum request size of 2000 MB and no time limit. But we should do some big testing, I am sure that in some parts of FREME large requests will break the system.

**Comment by jnehring** [comment]
@ArneBinder please try to submit large files to FREME NER. Create textfiles of 50 MB, 100 MB, 500 MB and 1 GB size and try to submit them to FREME NER on freme-dev.

**Comment by ArneBinder** [comment]
A request to FREME NER with 500KB already raises:
```
{
  "exception": "eu.freme.broker.exception.ExternalServiceFailedException",
  "path": "/e-entity/freme-ner/documents",
  "error": "Bad Gateway",
  "status": 502,
  "timestamp": 1452014214010
}
```

50MB:
```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>502 Bad Gateway</title>
</head><body>
<h1>Bad Gateway</h1>
<p>The proxy server received an invalid
response from an upstream server.<br />
</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at api-dev.freme-project.eu Port 80</address>
</body></html>
```
3.3MB of turtle sent to e-Link or the filter controller produces the same.

**Comment by jnehring** [comment]
Please try large requests for
- FREME NER 
- e-Translation
- NIF converter

The results of this test may influence [FREME configuration documentation](http://api-dev.freme-project.eu/doc/knowledge-base/freme-for-sysadmins/configuration-options.html) multipart.maxFileSize
and multipart.maxRequestSize
